prod.yaml.jinja

```diff
@@ -102,6 +103,19 @@ services:
         project_name,
       ) }}
     {%- endif %}
+  {%- else %}
+   db:
+    extends:
+      file: common.yaml
+      service: db
+    env_file:
+      - .docker/db-creation.env
+    restart: unless-stopped
+    {%- if postgres_exposed %}
+    ports:
+      - "{{ exposed_port }}:5432"
```

Comment:
Why repeating some part of the structure? And this should be only if port exposed, not on every case.